src/misc/TranslationKey.ts

```diff
@@ -1450,4 +1450,6 @@ export type TranslationKeyType =
 	| "yourFolders_action"
 	| "yourMessage_label"
 	| "you_label"
-	| "emptyString_msg"
\ No newline at end of file
+	| "emptyString_msg"
+	| "exportUsers_action"
+	| "importExport_action"
```

Comment:
I wouldn't add this key, we might add more things into that menu, maybe best just `more_label`